helix-core/src/test/java/org/apache/helix/integration/controller/TestOfflineNodeTimeoutDuringMaintenanceMode.java

```diff
@@ -0,0 +1,242 @@
+package org.apache.helix.integration.controller;
```

Comment:
Do we have an end-to-end integration test here, such as verify that a timeout-ed instance during maintenance mode will not get assigned any partitions (received any state transition messages)?

Reply:
I will cover that in a CHO test.

Reply:
Updated.